test/tests/components/image-builder/builder_test.go

```diff
@@ -44,12 +44,12 @@ func TestBaseImageBuild(t *testing.T) {
 					From: &imgapi.BuildSource_File{
 						File: &imgapi.BuildSourceDockerfile{
 							DockerfileVersion: "some-version",
-							DockerfilePath:    ".gitpod.Dockerfile",
+							DockerfilePath:    "test/tests/components/image-builder/test.Dockerfile",
 							ContextPath:       ".",
 							Source: &csapi.WorkspaceInitializer{
 								Spec: &csapi.WorkspaceInitializer_Git{
 									Git: &csapi.GitInitializer{
-										RemoteUri:  "https://github.com/gitpod-io/dazzle.git",
+										RemoteUri:  "https://github.com/gitpod-io/gitpod.git",
```

Comment:
It doesn't pull gitpod here, instead it pulls a small dockerfile: test/tests/components/image-builder/test.Dockerfile
since dazzle gitrepo is using workspace-full image which is big and heavy.